README.md

```diff
@@ -39,21 +39,17 @@ Later Raoul Diffou joined to take over as project manager as Thomas and Bryan ha
 * [Imperative vs Declarative Cucumber](http://fasteragile.com/blog/2015/01/19/declarative-user-stories-translate-to-good-cucumber-features/)
 * [JavaScript Acceptance test trials](https://bibwild.wordpress.com/2016/02/18/struggling-towards-reliable-capybara-javascript-testing/)
 
-## Relevant rake tasks
-
-```bash
-rake fetch_github_last_updates
-rake fetch_github_languages
-rake fetch_github_content_for_static_pages
-rake fetch_github_readme_files
-rake fetch_github_commits
-rake geocode:all
-rake mailer:send_welcome_message
-rake modify_event_participation
-rake paypal:create_paypal_plans
-rake stats
-rake user:create_anonymous
-rake vcr_billy_caches:reset
-```
-
-Updating the pages requires the administrator to run the `rake fetch_github:content_for_static_pages` task.
+## Walkthroughs
```

Comment:
Is this change from an old version of this file?  I don't think this file needs to be changed for this PR.